web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -128,6 +128,8 @@ function isR2L() {
   });
 })(jQuery);
 
+fixmystreet.hooks = fixmystreet.hooks || {};
```

Comment:
What I meant was that this line would be in `staff.js` and then it would be self-contained and not overwrite any other hooks created in separate files. As we know this file comes before `staff.js` this doesn't matter, but sorry if I was unclear in what I was trying to explain.